### DISCUSSION TITLE
LinkedIn Social Auth in AdonisJS 5

### DISCUSSION THREAD

**Discussion by derwydd** [adonisjs/core#2564] [General]
What is the timeframe to add LinkedIn Auth?

**Comment by iagobruno** [comment]
Same for Facebook Auth 👀

**Comment by LeadcodeDev** [comment]
Hello !
It's true that having the provider in the Ally module would be a big plus.

If this feature is planned, you can create your own provider or use an existing npm module!

Adonis is nothing more or less than javascript! You don't have to lock yourself in a framework ! :)

**Reply by thetutlage** [reply]
It is quite easy to add your own custom providers. I am working on the docs on same and it will be out soon

**Reply by LeadcodeDev** [reply]
I can't wait ! :sunglasses:
